.eslintrc

```diff
@@ -8,8 +11,13 @@
 
   "rules": {
     "no-mixed-operators": [2, { "allowSamePrecedence": true }],
+
     "jsx-a11y/no-static-element-interactions": 1, // TODO: enable
+
     "react/no-did-mount-set-state": 0, // necessary for server-rendering
+
+    "react-with-styles/no-unused-styles": 2,
```

Comment:
do these come in with a recommended config at all, or only enabled one at a time?

Reply:
One of them is in recommended config. The other is not yet.

Reply:
Should we use the recommended config then, and only add the rule that differs?

Reply:
Sure thing, I'll push an update shortly.